docs/tasks/stateful-sets/deleting-a-statefulset.md

```diff
@@ -0,0 +1,102 @@
+---
+assignees:
+- bprashanth
+- erictune
+- foxish
+- janetkuo
+- smarterclayton
+
+---
+
+{% capture overview %}
+This task shows you how to delete a Stateful Set.
+{% endcapture %}
+
+{% capture prerequisites %}
+
+* If you are not using Stateful Sets, you can skip this task now. 
+
+{% endcapture %}
+
+{% capture steps %}
+
+### Deleting a StatefulSet
+
+A Stateful Set can be deleted like other resources in kubernetes. 
+
+```shell
+$ kubectl delete -f statefulset.yaml 
+statefulset "web" deleted
+
+```
+
+
+```shell
+$ kubectl delete statefulsets web
+statefulset "web" deleted
+
+```
+
+The associated headless service may need to be deleted separately after the Stateful Set itself is deleted.
+
+```shell
+$ kubectl delete service web
+service "web" deleted
+
+```
+
+Deleting a Stateful Set through kubectl will scale it down to 0, thereby deleting all pods that are a part of it. If you wish to delete just the Stateful Set and not the pods, please use --cascade=false.
+
+```shell
+$ kubectl delete -f statefulset.yaml --cascade=false
+petset "web" deleted
+
+$ kubectl get po -l app=myapp
+NAME      READY     STATUS    RESTARTS   AGE
+web-0     1/1       Running   0          21h
+web-1     1/1       Running   0          21h
+
+$ kubectl delete po -l app=myapp
+pod "web-0" deleted
+pod "web-1" deleted
+```
+
+#### Persistent Volumes
+
+Deleting the pods will *not* delete the volumes. Until we finalize the recycle policy for these volumes they will have to get cleaned up by an admin. This is to ensure that you have the chance to copy data off the volume before deleting it. Simply deleting the PVC after the pods have left the [terminating state](/docs/user-guide/pods/index#termination-of-pods) should trigger deletion of the backing Persistent Volumes. 
```

Comment:
Good catch. It also says not to use [passive voice](http://kubernetes.io/docs/contribute/style-guide/#use-active-voice), which makes it that much harder to avoid "we".

However, given that we also are not supposed to [make statements about the future](http://kubernetes.io/docs/contribute/style-guide/#avoid-statements-about-the-future), I would argue for just removing the entire sentence "Until we finalize ...".